lib/chef/resources.rb

```diff
@@ -139,4 +140,4 @@
 require "chef/resource/windows_printer_port"
 require "chef/resource/windows_shortcut"
 require "chef/resource/windows_task"
-require "chef/resource/locale"
```

Comment:
This part needs to go back in. I think this was removed during the rebase

Reply:
I moved it to line 60 (I hope)

Reply:
Oops. Yep it's still there